### PR TITLE
chore: added contract example to the readme, added license

### PR DESCRIPTION
**Pull request by denismaxim0v**
Draft for now. The license is AGPL as agreed. Please add yourselves to MAINTAINERS.md - First Last (email / handle)

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Maintainers
```
Comment (mradkov) [inline MAINTAINERS.md:1]:
```suggestion
# Maintainers
Adam Avramov (adam@hack.bg / @egasimus)
Asparuh Kamenov (asparuh@hack.bg / @aakamenov)
Denis Maximov (denis@hack.bg / @denismaxim0v)
Milen Radkov (milen@hack.bg / @mradkov)
```

Comment (mradkov) [reply MAINTAINERS.md:1]: Something like this?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,661 @@
...
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea of what it does.>
+    Copyright (C) <year>  <name of author>
```
Comment (mradkov) [inline LICENSE:633]: Should these be filled in now?

File: MAINTAINERS.md (hunk 1 above)
Comment (denismaxim0v) [reply MAINTAINERS.md:1]: Yeah, Perfect

**Review by mradkov — APPROVED** [review]

**Pull request merged**